training/data/data_06_14_19.json

```diff
@@ -0,0 +1,3403 @@
+{
```

Comment:
mongodb stores data in json format. shouldn't we use that in place of a raw json file ?